### PR TITLE
Add Hacktoberfest section + articles

### PR DESCRIPTION
**Pull request by mishmanners**
## What does this PR do?

Add a section in the README for Hacktoberfest 2022
Add articles and video

## Related PRs and Issues

https://github.com/appwrite/awesome-appwrite/issues/89
https://github.com/appwrite/appwrite/issues/1774

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Sure have! Happy days 🌈

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,10 @@ The Almost Netflix series is a tutorial for building a Netflix clone with Appwri
 * [What is Appwrite and how to contribute for Hacktoberfest](https://dev.to/mishmanners/four-projects-to-contribute-to-this-hacktoberfest-pep)
 * [Appwrite, giving it a shot this Hacktoberfest ‘21](https://medium.com/@anand26shweta/appwrite-giving-it-a-shot-ad1ef79ae44f)
 
+## Hacktoberfest 2022
+* [Appwrite, Hacktoberfest, and how to contribute, including AMA with viewers](https://www.twitch.tv/videos/1634653922)
+* [Featured on the GitHub Blog](https://github.blog/release-radar-sept-2022)
```
Comment (stnguyen90) [inline README.md:107]: I get a

> Whoops, we haven't written that blog post yet!

error when trying to browse to this.

File: README.md [hunk 2]
```diff
@@ -102,6 +102,10 @@ The Almost Netflix series is a tutorial for building a Netflix clone with Appwri
 * [What is Appwrite and how to contribute for Hacktoberfest](https://dev.to/mishmanners/four-projects-to-contribute-to-this-hacktoberfest-pep)
 * [Appwrite, giving it a shot this Hacktoberfest ‘21](https://medium.com/@anand26shweta/appwrite-giving-it-a-shot-ad1ef79ae44f)
 
+## Hacktoberfest 2022
+* [Appwrite, Hacktoberfest, and how to contribute, including AMA with viewers](https://www.twitch.tv/videos/1634653922)
```
Comment (stnguyen90) [inline README.md:106]: Sorry, what part of this video is related to Appwrite?

File: README.md [hunk 3]
```diff
@@ -102,6 +102,10 @@ The Almost Netflix series is a tutorial for building a Netflix clone with Appwri
 * [What is Appwrite and how to contribute for Hacktoberfest](https://dev.to/mishmanners/four-projects-to-contribute-to-this-hacktoberfest-pep)
 * [Appwrite, giving it a shot this Hacktoberfest ‘21](https://medium.com/@anand26shweta/appwrite-giving-it-a-shot-ad1ef79ae44f)
 
+## Hacktoberfest 2022
```
Comment (stnguyen90) [inline README.md:105]: I don't think we want a Hacktoberfest specific section in this readme.

File: README.md (hunk 1 above)
Comment (mishmanners) [reply README.md:107]: it should be there now. It was caching through from the WP side. Can you check it now 😄

File: README.md (hunk 2 above)
Comment (mishmanners) [reply README.md:106]: In the middle bit where we featured AppWrite and showed some of the repos to contribute to during Hacktoberfest.

File: README.md (hunk 3 above)
Comment (mishmanners) [reply README.md:105]: There is one for 2021... so maybe just combine them both?

File: README.md (hunk 2 above)
Comment (stnguyen90) [reply README.md:106]: Would you please provide the timestamp?

Comment (mishmanners) [reply README.md:106]: I'll find it for you

File: README.md (hunk 1 above)
Comment (mishmanners) [reply README.md:107]: @stnyugen90 can you please check now.

**Review by mishmanners — COMMENTED** [review]
Upon further inspection, I think let's take out the live stream. I mentioned it a bit, but not enough. Leave the blog in as that's awesome. Ready for review now.

**Comment by stnguyen90** [comment]
Great PR! I've approved it and we'll merge it soon!

**Review by stnguyen90 — APPROVED** [review]

**Comment by mishmanners** [comment]
Sweet thank you!

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**